packages/components/src/components/entities/models.ts

```diff
@@ -225,7 +225,8 @@ export class EntityIdCreationModel extends Record({
     selectionKey: undefined,
     targetEntityType: undefined,
     entityCount: 0,
-    entityDataType: undefined
+    entityDataType: undefined,
+    auditBehavior: undefined
```

Comment:
I see that none of these `auditBehavior` props have default values on the shared components side, which I think makes sense. I assume if undefined is passed through the insert/update/delete APIs then it will just default to the table's audit behavior, right?

Reply:
Yes, we'll default to the time before this property ever existed.